handlers/viber/keyboard.go

```diff
@@ -0,0 +1,116 @@
+package viber
+
+import (
+	"fmt"
+	"html"
+	"strings"
+	"unicode/utf8"
+)
+
+// KeyboardButton is button on a keyboard, see https://developers.viber.com/docs/tools/keyboards/#buttons-parameters
+type KeyboardButton struct {
+	ActionType string `json:"ActionType"`
+	ActionBody string `json:"ActionBody"`
+	Text       string `json:"Text"`
+	TextSize   string `json:"TextSize"`
+	Columns    string `json:"Columns,omitempty"`
+	BgColor    string `json:"BgColor,omitempty"`
+}
+
+// Keyboard models a keyboard, see https://developers.viber.com/docs/tools/keyboards/#general-keyboard-parameters
+type Keyboard struct {
+	Type          string           `json:"Type"`
+	DefaultHeight bool             `json:"DefaultHeight"`
+	Buttons       []KeyboardButton `json:"Buttons"`
+}
+
+const (
+	// maxColumns refer to the maximum number of columns units in a row
+	maxColumns = 6
+	// maxRowsRunes refer to maximum number of runes in the same row to help define buttons layout
+	maxRowRunes = 38
+	// paddingRunes to help to calculate the size of button column width
+	paddingRunes = 2
+)
+
+var textSizes = map[string]string{"small": "small", "regular": "regular", "large": "large"}
```

Comment:
if you're using a map to check something is in a set, use `map[string]bool`

Reply:
Sure, i got it. Thanks.